src/rescle.cc

```diff
@@ -11,6 +11,8 @@
 #include <atlstr.h>
 #include <sstream> // wstringstream
 #include <iomanip> // setw, setfill
+#include <iostream>
+#include <string>
```

Comment:
Are these new includes needed? Looks like `string` is included in the `.h` file already